forge-gui/res/cardsfolder/upcoming/fear_of_change.txt

```diff
@@ -0,0 +1,11 @@
+Name:Fear of Change
+ManaCost:G U
+Types:Enchantment Creature Nightmare
+PT:2/3
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Destination$ Battlefield | OptionalDecider$ You | Execute$ TrigExile | TriggerDescription$ When this creatures enters or dies, exile another creature you control. If you do, conjure a duplicate of a random creature card with mana value X onto the battlefield, where X is 2 plus the exiled creature's mana value.
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Destination$ Graveyard | OptionalDecider$ You | Execute$ TrigExile | Secondary$ True | TriggerDescription$ When this creatures enters or dies, exile another creature you control. If you do, conjure a duplicate of a random creature card with mana value X onto the battlefield, where X is 2 plus the exiled creature's mana value.
+SVar:TrigExile:DB$ ChangeZone | ValidTgts$ Creature.Other+YouCtrl | TgtPrompt$ Exile another creature you control. | Origin$ Battlefield | Destination$ Exile | SubAbility$ DBNameCard
```

Comment:
doesn't target

Reply:
Updated! Took inspiration from [Yannik, Scavenging Sentinel](https://scryfall.com/search?as=grid&order=name&q=oracle%3A%22exile+another+creature+you+control%22+%28game%3Apaper%29)